force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -300,15 +400,21 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Map<String, Integer> addItemsToMap() {
         // Initialize an empty Map
+        Map<String, Integer> mapItems = new Map<String, Integer>();
+        List<String> fruits = new List<String>{ 'Apples', 'Oranges', 'Bananas', 'Pears', 'Grapes'};
+        List<Integer> amounts = new List<Integer>{ 5, 10, 15, 20, 25 };
 
         // Add key-value pairs to the Map
         // add Apples => 5
         // add Oranges => 10
         // add Bananas => 15
         // add Pears => 20
         // add Grapes => 25
+        for (Integer x = 0; x < 5; x++) {
+            mapItems.put(fruits[x], amounts[x]);
+        }
 
-        return null; // Replace null with the variable you used to store the result
+        return mapItems; // Replace null with the variable you used to store the result
```

Comment:
Probably a little harder than it needed to be. I would suggest either using the Map constructor or Map.put('Apples', 5)

No iteration is needed to solve this one.